src/main/java/duke/Events.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Events class will handle tasks categorised as event.
+ */
+public class Events extends Task {
```

Comment:
I noticed that you have named the subclasses in plural form (e.g. Events, Todos, Deadlines) but they represent only 1 Task object rather than a list of event/todo/deadline tasks for example. Perhaps you can consider this point in future for clarity.